CardPayments/src/main/java/com/paypal/android/cardpayments/CardClient.kt

```diff
@@ -72,6 +76,18 @@ class CardClient internal constructor(
         }
     }
 
+    /**
+     * Initiate vaulting for a Card payment method.
+     *
+     * NOTE: 3DS Challenge will occur when required.
+     */
+    fun vault(vaultRequest: VaultRequest) {
```

Comment:
Since we have merchants using the SDK now, can we make the vault w/o purchase flow (for now) internal only until it's fully functional?

Similar question for the vault w/ purchase flow. Is that ready to be public? I know we still need to add to our `docs/` directory for that flow.